src/components/views/SettingsView/index.tsx

```diff
@@ -53,6 +57,26 @@ const SettingsView = () => {
         ],
         preview: PREVIEW.SERVICE,
       }),
+      {
```

Comment:
You're gonna want to update the dependencies in the memo array. I tried this out locally and the menu wasn't updating because `deviceColor` wasn't added 😛

Reply:
Yeah, gotta add eslint to this repo :P